docs/_future/manifest.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Manifest
+lang: en
+slug: manifest
+order: 2
+layout: future
+---
+
+
+<div class="section-content">
+
+Your project should contain a `manifest.json` file that defines your app's manifest, which is where you can configure your application name and scopes, declare the functions your app will use, and more.
+Refer to the <a href="https://api.slack.com/reference/manifests" target="_blank">App Manifest documentation</a> to learn
+about the available manifest configurations.
+
+Notably this file informs Slack of the definitions for:
+
+* [Functions](/bolt-python/future/concepts#manifest-functions)
```

Comment:
Thoughts on including a very brief, one-line definition of a Function and a Workflow within these bullets?

Reply:
I feel like this would be super-fluent since the link brings them directly the section on the topic 🤔

Reply:
ahh yeah, I see what you're saying! I think specifically for Functions, I wasn't sure if it was defined clearly enough throughout these docs. If it feels like the definition for that is pretty explicit though in itself, then we can definitely leave it as it!

Reply:
I think that these docs should be sufficient enough for now, the `manifest documentation` will definitely be revised in the very near future